views/partials/signUpForm.hbs

```diff
@@ -0,0 +1,99 @@
+<form class="form-signup" action="/submit" method="POST">
+    <h2 class="form-signup-heading">Not still a member, then sign up</h2>
+    <label for="name" class="sr-only">Name</label>
+    <input type="text" id="name" class="form-control" placeholder="Name" required>
+    <label for="signupEmail" class="sr-only">Email address</label>
+    <input type="email" id="signupEmail" class="form-control" placeholder="Email address" required>
+    <label for="dob">Email address</label>
+    <div class="dob__wrapper js-dob__wrapper">
+        <select class="form-control date">
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+            <option value="11">11</option>
+            <option value="12">12</option>
+            <option value="13">13</option>
+            <option value="14">14</option>
+            <option value="15">15</option>
+            <option value="16">16</option>
+            <option value="17">17</option>
+            <option value="18">18</option>
+            <option value="19">19</option>
+            <option value="20">20</option>
+            <option value="21">21</option>
+            <option value="22">22</option>
+            <option value="23">23</option>
+            <option value="24">24</option>
+            <option value="25">25</option>
+            <option value="26">26</option>
+            <option value="27">27</option>
+            <option value="28">28</option>
+            <option value="29" class="gt28">29</option>
+            <option value="30" class="gt28">30</option>
+            <option value="31" class="gt28 gt30">31</option>
+        </select>
+        <select class="form-control month">
+            <option value="1">Jan</option>
+            <option value="2">Feb</option>
+            <option value="3">Mar</option>
+            <option value="4">Apr</option>
+            <option value="5">May</option>
+            <option value="6">Jun</option>
+            <option value="7">July</option>
+            <option value="8">Aug</option>
+            <option value="9">Sept</option>
+            <option value="10">Oct</option>
+            <option value="11">Nov</option>
+            <option value="12">Dec</option>
+        </select>
+        <select class="form-control year">
+            <option value="2012">2012</option>
```

Comment:
Same over here. If you are using calendar type functionality, checkout native input type="date".